spec/fabricators/account_moderation_note_fabricator.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 Fabricator(:account_moderation_note) do
-  content 'MyText'
+  content { Faker::Lorem.sentences }
```

Comment:
I think we should probably avoid hard coded strings, since we could encounter brittle tests through them.

Reply:
I feel the opposite here -- we should strongly prefer known/consistent values over random values in specs. We should be working to slowly remove faker from this sort of randomness-introducing usage. If there's some edge case to cover, we can add explicit values that capture that edge case instead of hoping faker might find it.

Related but separately - there's an argument to be made that we should (unrelated to this feature, just a generic dev idea...) work on some better more representative set of "dev data" which we could load into DB to have more plausible local dev env values, which might include sending more plausible/useful attributes through the fabricators. I could take a stab at this once the team catches up on PR review a bit.

Reply:
I generally go with "we want random data to ensure we've not an implicit validation in a test", where something passes by chance, but wouldn't actually pass in usage.

Reply:
If I understand that concern correctly, there actually is a spec that loads all the fabricators and ensures they can be saved w/out validation errors and is trying to catch exactly that issue.

If you mean something else ... I'm not sure.

Reply:
So if we have a hard coded string, that value might not be "realistic" with how users actually interact with that model, hence preferring fuzzing here.